File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLSubMenuBarProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLSubMenuBarProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> Data;
        public String ParentMenuWindowID;
        public String ParentIndexInParentMenu;
        public List<Object> ChildMenuWindowIDs;
        public String DropDownColorStart;
        public String DropDownColorEnd;
        public Object Tag;
        public String TabStopIndex;

        CCLSubMenuBarProps()
        {
            Data = new ArrayList<Object>();
            ChildMenuWindowIDs = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLSubMenuBarProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLSubMenuBarProps
    {
        public class MenuItem
        {
            public string Name { get; set; }
            public string FontColor { get; set; }
            public int FontHeight { get; set; }
            public string FontName { get; set; }
            public int HasSubMenu { get; set; }
            public string Function { get; set; }
            public List<MenuItem> MenuItems { get; set; }
        }

        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<MenuItem> Data { get; set; }
        public int ParentMenuWindowID { get; set; }
        public int ParentIndexInParentMenu { get; set; }
        public List<int> ChildMenuWindowIDs { get; set; }
        public string DropDownColorStart { get; set; }
        public string DropDownColorEnd { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLSubMenuBarProps()
        {
            Data = new List<MenuItem>();
            ChildMenuWindowIDs = new List<int>();
        }
    }
}